README.md

```diff
@@ -1,16 +1,18 @@
 # About Stubby
 
 Stubby is an application that acts as a local **DNS Privacy stub resolver** (using DNS-over-TLS). Stubby encrypts DNS queries sent 
-from a client machine (desktop or laptop) to a DNS Privacy resolver increasing end user privacy. Stubby is in the early stages 
+from a client machine (desktop or laptop) to a DNS Privacy resolver(s) increasing end user privacy. Stubby is in the early stages 
 of development but is suitable for technical/advanced users. A more generally user-friendly version is on the way!
 
 Stubby provides DNS Privacy by:
 
 * Running as a daemon
-* Listening on the loopback address to send all outgoing DNS queries received on that address out over TLS
+* Listening on the loopback address to send all outgoing DNS queries received on that address out over TLS (1.2 or 1.3)
 * Using a default configuration which provides Strict Privacy and uses a subset
 of the available [DNS Privacy servers](https://dnsprivacy.org/wiki/x/E4AT)
 
+Note: Resolution of DNS queries using TLS is typically several times slower than standard UDP based DNS, this may (but not necessarily) make a noticable difference to system performance.
```

Comment:
Hi Inudge,

This is not necessarily true.  Once the TLS session is set up, the should not be a difference round trip-wise and could even be faster when multiple requests are pipe-lined (and talking to an upstream that supports Out Of Order responses)